src/libslic3r/PrintObject.cpp

```diff
@@ -481,6 +470,18 @@ void PrintObject::prepare_infill()
     this->discover_horizontal_shells();
     m_print->throw_if_canceled();
 
+    // this will detect bridges and reverse bridges
```

Comment:
Any reason to move this code block?

Reply:
yeap, it fixes an issue with bridging over small holes that I found while making this PR. 

If processed horizontal shells is done after the external surfaces processing, when the ensure vertical thickness option is disabled the bridges dont expand like when the ensure vertical shell thickness option is turned on. This results in the bridges collapsing inside small holes, like the bottom bridge layer of the benchy.

Before moving the code block:
![image](https://github.com/SoftFever/OrcaSlicer/assets/59056762/7799cee8-0cb7-4be8-8c9a-7e77b195179f)
After moving the code block:
![image](https://github.com/SoftFever/OrcaSlicer/assets/59056762/dc866ffb-8684-44cd-aa01-567e423bcec5)
With vertical thickness enabled:
![image](https://github.com/SoftFever/OrcaSlicer/assets/59056762/d2aed6c1-8863-4303-9f8a-79f1a4c31edb)